sample/src/main/java/com/timehop/stickyheadersrecyclerview/sample/MainActivity.java

```diff
@@ -35,7 +34,7 @@ protected void onCreate(Bundle savedInstanceState) {
     final ToggleButton isReverseButton = (ToggleButton) findViewById(R.id.button_is_reverse);
 
     // Set adapter populated with example dummy data
-    final SampleArrayHeadersAdapter mAdapter = new SampleArrayHeadersAdapter();
+    final AnimalsHeadersAdapter mAdapter = new AnimalsHeadersAdapter();
```

Comment:
I realize you didn't do this but since you're working on this could you please change `mAdapter` to `adapter`? Not sure how that slipped in :)